src/ui/SessionExpiryTracker/SessionExpiryTracker.tsx

```diff
@@ -64,18 +64,32 @@ const SessionExpiryTracker: React.FC = () => {
     }
   }, [sessionExpiryTimeString, checkSession, setupSessionIntervalCheck])
 
+  useEffect(() => {
+    if (!redirectToLogout) {
+      return
+    }
+
+    const handleLogout = async () => {
+      await fetch(`${config.API_URL}/logout`, {
+        method: 'POST',
+        credentials: 'include',
+      })
+      Cookies.remove(COOKIE_SESSION_EXPIRY)
+      localStorage.setItem(LOCAL_STORAGE_SESSION_EXPIRED_KEY, 'true')
+      localStorage.removeItem(LOCAL_STORAGE_SESSION_TRACKING_KEY)
+    }
+
+    handleLogout()
+  }, [redirectToLogout])
+
   if (!sessionExpiryTimeString || !redirectToLogout) {
     return null
   }
 
-  Cookies.remove(COOKIE_SESSION_ID, { path: '' })
-  localStorage.setItem(LOCAL_STORAGE_SESSION_EXPIRED_KEY, 'true')
-  localStorage.removeItem(LOCAL_STORAGE_SESSION_TRACKING_KEY)
-
   return config.IS_SELF_HOSTED ? (
-    <Redirect to={'/'} />
+    <Redirect to={'/?session=expired'} />
   ) : (
-    <Redirect to={'/login'} />
+    <Redirect to={'/login?session=expired'} />
```

Comment:
Thoughts on using our query string helper here to set these, a bit more explicit and also adds in browser compat in case there are any issues